js/script.js

```diff
@@ -0,0 +1,134 @@
+/*  Anna Sullivan
+    script.js
+    INFO2134WW
+    Thoendel
+    5/2/2020
+*/
+window.addEventListener('load', () => {//event listener on page load
+    
+    startClock();
+        //YOUR CODE SHOULD START BELOW THIS LINE
+    
+    navigator.geolocation.getCurrentPosition(findLocation);//call getCurrentPosition using navigator and geolocation
+        
+    function findLocation(position){//function to find position of user
+       console.log(`Latitude: `+ position.coords.latitude `, Longitude: `+ 
+       position.coords.longitude +`.`);
+    }
+    
+    const buttons = document.getElementsByTagName('button'); //const set to the button tag name
+    
+    for(const button of buttons){//for loop button const
+        const buttonID = button.dataset.id; //const buttonID set to button id
+    button.addEventListener('click', () => {//event listen on click of button
+        if(buttonID == 'getCurrentWX'){//if buttonID = getCurrentWX call function getCurrentWX
+            getCurrentWX(); 
+        }
+        else if(buttonID == 'getFiveDay') {//if buttonID = getFiveDay call function getFiveDay
+                getFiveDay();
+            }
+        })
+    }
+    
+    function getCurrentWX() {//function getCurrentWX
+        console.log(`getCurrentWX() invoked`);
+        let url1 = `api.openweathermap.org/data/2.5/weather?` + `lat=` + position.coords.latitude +
+        `& lon= ` + position.coords.longitude + `appid=c4b2524f1d913f75dbba99b34cf5341f`;//appid to fetch position info

```

Comment:
again you are mixing template and string syntax. This won't work - and I can't test farther as it isn't working. I'll give you through step 4.